src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -211,24 +212,52 @@
                 :func:`google.gax.construct_settings` for the structure of
                 this data. Falls back to the default config if not specified
                 or the specified config is missing data points.
-              app_name (string): The codename of the calling service.
-              app_version (string): The version of the calling service.
+              app_name (string): The name of the application calling
+                the service. Recommended for analytics purposes.
+              app_version (string): The version of the application calling
+                the service. Recommended for analytics purposes.
+              lib_name (string): The API library software used for calling
+                the service. (Unless you are writing an API client itself,
+                leave this as default.)
+              lib_version (string): The API library software version used
+                for calling the service. (Unless you are writing an API client
+                itself, leave this as default.)
+              metrics_headers (dict): A dictionary of values for tracking
```

Comment:
One problem is that the GAPICs are using this to pass info to GAX, but there will later be A/B headers, which go at the end. Right now if we add those tomorrow in Python, they will be out of order.

The place to fix that is in GAX.